<?xml version="1.0"?>
<launch>
        <arg name="route_file_folder" default="/opt/carma/routes/" />
	<node pkg="route" type="route" name="route" output="screen">
        <param name="route_file_path" type="str" value="$(arg route_file_folder)" />
        </node>
</launch>
